libs/layout/src/lib/directives/drag-and-drop/drag-and-drop-item.directive.ts

```diff
@@ -0,0 +1,258 @@
+import {
+	Directive,
+	ElementRef,
+	EventEmitter,
+	HostBinding,
+	HostListener,
+	Input,
+	Output,
+	signal,
+	WritableSignal,
+} from '@angular/core';
+import { CdkDropList } from '@angular/cdk/drag-drop';
+
+import { NgxAccessibleDragAndDropAbstractService } from '../../abstracts';
+import { NgxAccessibleDragAndDropMoveEvent, NgxAccessibleDragAndDropMoveType } from '../../types';
+import { NgxHasFocusDragAndDropDirective } from './has-focus.directive';
+import { NgxAccessibleDragAndDropContainerDirective } from './drag-and-drop-container.directive';
+import { NgxAccessibleDragAndDropHostDirective } from './drag-and-drop-host.directive';
+
+/**
+ * A directive to handle accessible drag and drop flows. This directive is meant to be placed on the item(s) of the drag and drop container(s).
+ */
+@Directive({
+	selector: '[ngxAccessibleDragAndDropItem]',
+	exportAs: 'ngxAccessibleDragAndDropItem',
+	standalone: true,
+	host: {
+		'[attr.tabIndex]': 'tabIndex()',
+	},
+})
+export class NgxAccessibleDragAndDropItemDirective extends NgxHasFocusDragAndDropDirective {
```

Comment:
I'm gonna be honest, I don't really like the fact that `Accessible` is always visible in the services, directives and types. Yes, we focus on the a11y, but should it be this explicitly mentioned in each name?

Reply:
I mean, the sole purpose of these directives, types and services is accessibility... They serve no other purpose than that; I don't understand why we'd not include it in the name? 😅 

It's not like we do this with everything we do, this is just 100% intended for accessible drag and drop, I think the intention should be clear here.